java/client/src/main/java/glide/api/commands/BitmapBaseCommands.java

```diff
@@ -73,4 +73,25 @@ public interface BitmapBaseCommands {
      * }</pre>
      */
     CompletableFuture<Long> bitcount(String key, long start, long end, BitmapIndexType options);
+
+    /**
+     * Sets or clears the bit at <code>offset</code> in the string value stored at <code>key</code>.
+     * The <code>offset</code> is zero-based index, with <code>0</code> being the first element of the
+     * list, <code>1</code> being the next element, and so on. The <code>offset</code> must be less
+     * than <code>2^32</code> and greater than or equal to <code>0</code>. If a key is non-existent
+     * then the bit at <code>offset</code> is set to <code>0</code> or <code>1</code> and the
+     * preceding bits are set to <code>0</code>.
+     *
+     * @see <a href="https://redis.io/commands/setbit/">redis.io</a> for details.
+     * @param key The key of the string.
+     * @param offset The index of the bit to be set.
+     * @param value The bit value to set at <code>offset</code>.
```

Comment:
Do you think we should clarify that this must be 0 or 1, or do you think this is obvious enough that it isn't needed? The variable type is long so users could end up passing other values

Reply:
Is there a reason we're returning Long instead of... say... an Int or [Byte](https://docs.oracle.com/javase/8/docs/api/java/lang/Byte.html)

Reply:
I thought we only had Long and Double types for all number types. Would introducing Integer/Byte/Short cause any issues?

@aaron-congo I added more context, thanks!

Reply:
As per the discussion offline, we are going to use long. This can be switched in the future if a boolean or byte is preferred.